style.css

```diff
@@ -0,0 +1,220 @@
+*{
+    margin: 0;
+    padding: 0;
+    font-family: 'Poppins', sans-serif;
+}
+
+:root{
+    --primary-color: #1a406b;
+    --white-color : #fff;
+    --black-color : black;
+    --container-gb-color : #f2f2f2;
+    --facebook-bg-color :  #3b5998;
+    --twitter-bg-color :  #00acee;
+    --linkedin-bg-color :  #0e76a8;
+}
+
+/* sub-header */
+
+.sub-header{
+    display: flex;
+    justify-content: flex-end;
+    background-color: var(--primary-color);
+    width: 100%;
+    position: fixed;
+}
+
+.login-links {
+    gap: 5px;
+    margin-right: 3rem;
+    color: var(--white-color);
+}
+
+.login-links a{
+    text-decoration: none;
+    color: var(--white-color);
+}
+
+/* navbar */
+
+.navbar{
+    display: flex;
+    width: 100%;
+    justify-content: space-around;
+    align-items: center;
+    height: 4rem;
+    border-bottom: 1px solid;
+    gap: 30%;
+    background-color: var(--white-color);
+    position: sticky;
+    top: 1.5rem;
+}
+
+.nav-brand{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-direction: row-reverse;
+    gap: 1rem;
+}
+
+.nav-brand-name{
+    font-size: 1.8rem;
+    color: var(--primary-color);
+    font-weight: 600;
+    text-decoration: none;
+    cursor: pointer;
+}
+
+.nav-contents{
+    display: flex;
+    gap: 1.2rem;
+}
+
+.nav-brand-img{
+    height: 3rem;
+}
+
+.search-input{
+    height: 2rem;
+    min-width: 20rem;
+    max-width: auto;
+    padding-left: 1rem;
+}
+
+.search-input:hover{
+    font-weight: 600;
+}
+
+.cart {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+
+.cart-icon{
+    position: relative;
+    font-size: 1.5rem;
+}
+
+.cart-items-number{
+    position: absolute;
+    top: 10px;
+    right: 4.8rem;
+    height: 1.1rem;
+    width: 1.1rem;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    background-color: var(--primary-color);
+    color: var(--white-color);
+    border-radius: 50%;
+    z-index: 1;
+    font-size: 12px;
+}
+
+/* home page */
+
+.main-section{
+    padding: 0 4rem 0 4rem;
+}
+
+.img{
+    width: 100%;
+}
+
+.guide-container{
+    margin: 3rem 2rem 3rem 2rem;
+}
+
+.guide-content-container{
+    display: flex;
+    justify-content: space-around;
+    flex-direction: row-reverse;
+    background-color: var(--container-gb-color);
+    border-radius: 5px;
+}
+
+.guide-info{
+    padding: 4rem 6rem 4rem 7rem;
+}
+
+.guide-heading{
+    font-size: 2rem;
+}
+
+.guide-heading h1{
+    font-size: 3rem;
+}
+
+.guide-desc{
+    font-size: 1.1rem;
+    margin-bottom: 1.5rem;
+}
+.top-selling-cycle-container{
+    margin: 3rem 2rem 3rem 2rem;
+}
+
+.top-selling-cycles{
+    display: flex;
+    justify-content: space-around;
+    background-color: var(--container-gb-color);
+    border-radius: 5px;
+}
+
+.top-cycles-info{
+    margin-right: 8rem;
+    margin-top: 3rem;
+}
+
+.cycle-desc{
+    font-size: 1.8rem;
+}
+
+.cycle-price{
+    margin: 10px 0 1rem 0;
```

Comment:
use rem

Reply:
but this is less than 16px thats why I'm using px.

Reply:
but this is less than 16px thats why I'm using px.